### ISSUE TITLE
Failed request with fontshare font downloads

### ISSUE THREAD

**Issue by zzmrl** [feat-agency/vite-plugin-webfont-dl#28]
I am getting a request error when I try to download Fontshare fonts using this plugin in my sveltekit project.

![image](https://user-images.githubusercontent.com/26882268/230747182-7068f2c6-bf41-40aa-9765-697de9d9186c.png)

This is my exact `vite.config.ts`:

```ts
import { sveltekit } from '@sveltejs/kit/vite';
import Icons from 'unplugin-icons/vite';
import { ViteImageOptimizer } from 'vite-plugin-image-optimizer';
import webfontDownload from 'vite-plugin-webfont-dl';
import { defineConfig } from 'vitest/config';

export default defineConfig({
  css: {
    preprocessorOptions: {
      scss: {
        includePaths: [
          'node_modules/@bulvar/bulma',
          'node_modules/@bulvar/bulma/sass',
        ],
      },
    },
  },
  plugins: [
    sveltekit(),
    Icons({
      compiler: 'svelte',
    }),
    webfontDownload([
      'https://api.fontshare.com/v2/css?f[]=synonym@1&display=swap',
      'https://api.fontshare.com/v2/css?f[]=chillax@1&display=swap',
    ]),
    ViteImageOptimizer(),
  ],
  test: {
    include: ['src/**/*.{test,spec}.{js,ts}'],
  },
});
```

I've also tried:

```ts
export default defineConfig({
  plugins: [
    webfontDownload([
      'https://api.fontshare.com/v2/css?f[]=synonym@1&f[]=chillax@1&display=swap',
    ]),
  ]
})
```

And I tried placing `<link />` tags in my source as well with no luck.

I can try and get to the bottom of this when I have some time, but maybe it is more obvious to someone already working on it.

**Comment by 0xb4lint** [comment]
Hi @zmrl010!

Thanks for raising this issue!

I've just released [`v3.7.1-beta.4`](https://www.npmjs.com/package/vite-plugin-webfont-dl/v/3.7.1-beta.4) with [support for protocol relative URLs ](https://github.com/feat-agency/vite-plugin-webfont-dl/commit/3dfcbe98a56d341693ac2ea4e7b902badb203bac), could you please check it?

**Comment by zzmrl** [comment]
I can confirm that this update has fixed my issue! Thanks for publishing this fix so quickly!
